services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/unit_templates_manager/unitTemplateMinisTable.tsx

```diff
@@ -8,12 +8,12 @@ import { CONNECT, DIAGNOSTIC, GRAMMAR, LESSONS, PROOFREADER, READING_FOR_EVIDENC
 
 export const UnitTemplateMinisTable = ({ unitTemplates, userSignedIn }) => {
   const toolColors = {
-    [READING_FOR_EVIDENCE]: '#2C7F9B',
-    [CONNECT]: '#DF9E3D',
-    [DIAGNOSTIC]: '#EB4F47',
-    [GRAMMAR]: '#9035D6',
-    [PROOFREADER]: '#4D8DD9',
-    [LESSONS]: '#AD287B'
+    [READING_FOR_EVIDENCE]: '#2C7F9B', // $quill-teal
+    [CONNECT]: '#935B00', // $quill-gold-dark
+    [DIAGNOSTIC]: '#D3443D', // $quill-red
+    [GRAMMAR]: '#9035d6', // $quill-purple
+    [PROOFREADER]: '#3576C5', // $quill-blue
+    [LESSONS]: '#ad277b' // $quill-violet
```

Comment:
I could see us porting over all of color variables as constants into the Shared bundle in the future (not necessarily as part of these updates)